internal/util/util.go

```diff
@@ -0,0 +1,42 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+// GetEnvOrDefault returns environment variable value or if unset, a default value.
+func GetEnvOrDefault(key, defaultValue string) string {
```

Comment:
this is only used in internal/provider, just move it there, this means this package can then be renamed to podutil and the PodFromFile function can be renamed to `FromFile`

Reply:
Gotcha!

Reply:
Done!